src/aerialmap_display.cpp

```diff
@@ -592,19 +589,23 @@ void AerialMapDisplay::assembleScene()
   }
 
   // since not all tiles were loaded yet, this function has to be called again
-  if (!loadedAllTiles)
+  if (!all_tiles_loaded)
   {
     dirty_ = true;
   }
+  else
+  {
+    ROS_DEBUG_NAMED("rviz_satellite", "Finished assembling all tiles");
+  }
 
-  tileCache_.purge({ *lastCenterTile_, blocks_ });
+  tile_cache_.purge({ *center_tile_, blocks_ });
 
   checkRequestErrorRate();
 }
 
 void AerialMapDisplay::transformTileToMapFrame()
 {
-  if (not ref_fix_)
+  if (not ref_fix_ or not center_tile_)
   {
     ROS_FATAL_THROTTLE_NAMED(2, "rviz_satellite", "ref_fix_  not set, can't create transforms");
```

Comment:
`ref_fix_  not` -> `ref_fix_ not`

Reply:
I'm not getting that :/

Reply:
yeah my input was really badly worded. fixed it myself :)